runtime/help/plugins.md

```diff
@@ -31,48 +33,62 @@ of which there are 5 types:
 * Syntax header files
 
 In most cases, a plugin will want to add help files, but in certain
-cases a plugin may also want to add colorschemes or syntax files. It
-is unlikely for a plugin to need to add plugin files at runtime or
-syntax header files. No directory structure is enforced but keeping
-runtime files in their own directories is good practice.
+cases, a plugin may also want to add colorschemes or syntax files. It
+is unlikely for a plugin to need to add plugin or syntax header files at
+runtime. No directory structure is enforced, but keeping runtime files in their
+own directories is good practice.
 
 ## Lua callbacks
 
-Plugins use Lua but also have access to many functions both from micro
-and from the Go standard library. Many callbacks are also defined which
-are called when certain events happen. Here is the list of callbacks
-which micro defines:
+Plugins use Lua but also have access to many functions, both from micro
+and from the Go standard library. Plugins can also define functions that micro
+will call when certain events happen.
+Here is the list of callbacks that micro defines:
+
+* `init()`:
+    this function should be used for your plugin initialization. This function
+    is called after buffers have been initialized.
+
+* `preinit()`:
+    initialization function called before buffers have been initialized.
 
-* `init()`: this function should be used for your plugin initialization.
-   This function is called after buffers have been initialized.
+* `postinit()`:
+    initialization function called after `init()`.
 
-* `preinit()`: initialization function called before buffers have been
-   initialized.
+* `deinit()`:
+   cleanup function called when your plugin is unloaded, either manually by the
+   user, or by running the `reload` command.
```

Comment:
I'm ok with changing the style (function description beginning not at the same line but at the next line), but please use consistently the same indentation everywhere.

Also could you move these style changes (that are moving the text without changing its contents) to a separate commit, to separate them from the actual text changes? It's really hard to review when the two are mixed.

Reply:
Just in case, I didn't mean necessarily a separate PR. It can be done in one PR, just in separate commits.

Reply:
Good call! I was going to squash it but I guess I got distracted and forgot.

Reply:
Sorry for the delay, didn't realize you already squashed it. The damn github didn't send any notification.

Reply:
No worries at all! :)